### PR TITLE
GH-1068: As a smith I need to publish type definition packages (n4jsd) to NPM

### PR DESCRIPTION
**Pull request by mmews-n4**
In this PR:
- [x] change repo structure to have valid type definition projects which can be edited in the IDE without evaluation problems
- [x] remove obsolete fragment files

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+//webclients1-nexus.service.cd-dev.consul/repository/npm-internal/:_authToken=6fc043ce-9d5d-3c02-aa3b-234de4554823
```
Comment (yoosiba) [inline .npmrc]: checking in authorisation tokens into GH?

**Comment by yoosiba** [comment]
`n4jsd/npm/papaparse/4.5.0/@n4jsd/papaparse/` has no project file

File: .npmrc (hunk 1 above)
Comment (mmews-n4) [reply .npmrc]: I didn't realize that this file was checked in.

**Comment by dbo** [comment]
@mmews-n4 what about this PR?

**Comment by mmews-n4** [comment]
When this branch is merged, the old IDE won't work anymore (regarding the external libraries). So, whenever you updated the IDE, this can be merged.

**Pull request merged**